pallets/loans/src/weights.rs

```diff
@@ -58,14 +60,19 @@ pub trait WeightInfo {
     fn liquidate_borrow() -> Weight;
     fn add_reserves() -> Weight;
     fn reduce_reserves() -> Weight;
-    fn set_rate_model() -> Weight;
 }
 
 /// Weights for pallet_loans using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-    fn set_liquidation_incentive() -> Weight {
-        (3_787_000 as Weight).saturating_add(T::DbWeight::get().reads(1 as Weight))
+    fn active_market() -> Weight {
+        0
+    }
+    fn add_market() -> Weight {
+        0
+    }
+    fn update_market() -> Weight {
+        0
```

Comment:
IIRC, there is an automated benchmark system so these weights were initially set to 0

Reply:
I think you can run benchmark once to get those values

Reply:
Did a local run